hugo/content/documentation/administration/acl/index.md

```diff
@@ -0,0 +1,250 @@
+
+We may also define a group `userA` in `ChannelA1` and a group `userB` in `ChannelB` intending to only allow `userA` members to enter `ChannelA1` and its children, while only allowing `userB` members to enter the `ChannelB` channel.
+
+## ACL rules
+
+ACL rules are configured in the ACL tab of the channel edit dialog.
+
+![Screenshot of ACL dialog](screenshot-channel-acl.jpg)
+
+The Access Control *List* can have multiple rules. Rules may be inherited from parent channels - displayed in italics - or defined on this channel.
+
+Permission are evaluated from top to bottom. (*Allow* and *deny* may be overwritten further down.)
+
+### Selector
+
+A rule consists of a selector and permission actions.
+
+The **selector defines who the rule applies to** - a user, user group, or password group. It may use a prefix to indicate its type or modify its select-behavior.
+
+| Prefix | Prefix-Type   | Description                                      | Example                        |
+| ------ | ------------- | ------------------------------------------------ | ------------------------------ |
+| *none* | Id-Type       | User                                             | `User1`, `John Doe`            |
+| `@`    | Id-Type       | Group                                            | `@admin`, `@all`, `@moderator` |
+| `@#`   | Id-Type       | Channel password / Access Token                  | `@#secret`, `@#u8DhwG2`        |
+| `~`    | Eval-Locality | Evaluate in context of definition, not inherited | `@~sub 1,1`, `@~moderator`     |
+| `!`    | Inversion     | Invert membership (not in group)                 | `!TrustedUser`, `!@moderator`  |
+
+The tilde `~` eliminates any effect of inheritance or links.
+
+### Permission Allow/Deny
+
+The permission actions can allow or deny a permission. If the rule neither allows nor denies a permission it does not influence the top-to-bottom permission evaluation.
+
+### Rule Examples
+
+For example, given the following channel setup
+
+```text
+Root
+  ChanA1
+    ChanA11
+    ChanA12
+    ChanA13
+  ChanB
+```
+
+`@admin` defined on channel `Root` with *Allow* `Write ACL` will give members of the `admin` group full permissions.
+
+![Screenshot of giving admin group Write ACL permission](acl-admin-allow-writeacl.jpg)
+
+### Special Groups
+
+Some special groups can be used in ACL rules. The members of these groups are not defined explicitly. They are implicit - a consequence of the user state.
+
+| Group    | Description         | Notes          |
+| -------- | ------------------- | -------------- |
+| `@all`   | Applies to everyone                  | Useful for default permissions or for clearing complex rules to a well defined state. |
+| `@admin` | People with administrative authority |  |
+| `@in`    | In this channel                      |  |
+| `@out`   | Outside of this channel              |  |
+| `@sub`   | Sub-channel - `@sub,a,b,c`           |  |
+
+#### `in` Group

```

Comment:
Maybe we could combine this with the `out` meta group (which doesn't seem to have its own section anyway) as the given example (and any other that I could come up with) could also be solved by making use of the `out` group.

Reply:
I had difficulty understanding and testing the out group. Your suggestion of describing both in the same section sounds reasonable.

Reply:
I decided to keep them separate instead because I feel mixing the two would make the description less focused and obvious.

I chose to keep the out section short though, and mention that it is the inverse, with a short schematic example.